concourse/pipelines/templates/build_pipeline-tpl.yml

```
@@ -95,6 +99,12 @@ groups:
   - Test PXF-GP[[gp_ver]]-HDP2-SECURE-MULTI-NO-IMPERS on RHEL7
   - Test PXF-GP[[gp_ver]]-HDP2-Upgrade-Extension on RHEL7
 {% set gp_ver = None %}
+{% set gp_ver = 7 %}
+- name: PXF Tests - GP[[gp_ver]] on RHEL8
+  jobs:
+  - Test PXF-GP[[ gp_ver ]]-HDP2-SECURE-MULTI-IMPERS on RHEL8
+  - Test PXF-GP[[ gp_ver ]]-CLI on RHEL8
```

```suggestion
  - Test PXF-GP[[gp_ver]]-HDP2-SECURE-MULTI-IMPERS on RHEL8
  - Test PXF-GP[[gp_ver]]-CLI on RHEL8
```
for compactness and consistency